parser_test.go

```diff
@@ -5604,6 +5604,7 @@ func (checker *nodeTextCleaner) Enter(in ast.Node) (out ast.Node, skipChildren b
 		}
 	case *ast.FuncCallExpr:
 		node.FnName.O = strings.ToLower(node.FnName.O)
+		node.SetOriginTextPosition(0)
```

Comment:
What does this line work for?

Reply:
It's clear the text in the node, then we can use reflect.DeepEquals to compare two statements in :
https://github.com/pingcap/parser/blob/f416dc9329d0e340a892173f9628f5074d7077ca/parser_test.go#L388-L391